examples/javascript/cloudflare-worker/README.md

```diff
@@ -0,0 +1,36 @@
+# Logfire Cloudflare Worker Integration Example
+
+This example is based on the scaffolding created from `npm create cloudflare@latest`, and uses the [otel-cf-workers package](https://github.com/evanderkoogh/otel-cf-workers) to instrument a Cloudflare Worker and send traces and metrics to Logfire.
+
+## Run the project
+
+To run the example locally, you need a Logfire account and a project and a local installation of Node 20. If you don't have a Logfire account, create a free one [in Logfire](https://logfire.pydantic.dev/).
+
+Clone the repository:
+
+```bash
+git clone https://github.com/pydantic/logfire.git
+```
+
+`cd` into the example directory and install:
+
+```bash
+cd examples/javascript/cloudflare-worker
+npm install
```

Comment:
Installing gave me this output:

```
npm warn deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm warn deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.

added 200 packages, and audited 201 packages in 28s

39 packages are looking for funding
  run `npm fund` for details

3 vulnerabilities (1 moderate, 2 high)

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
```

It would be nice if it looked cleaner.